src/test/kotlin/no/nav/syfo/varsel/arbeidstaker/ArbeidstakerVarselWhenMultipleDialogmoteApiSpek.kt

```diff
@@ -0,0 +1,176 @@
+package no.nav.syfo.varsel.arbeidstaker
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.ktor.http.*
+import io.ktor.http.HttpHeaders.Authorization
+import io.ktor.server.testing.*
+import io.mockk.*
+import no.nav.syfo.application.mq.MQSenderInterface
+import no.nav.syfo.dialogmote.api.*
+import no.nav.syfo.dialogmote.api.domain.DialogmoteDTO
+import no.nav.syfo.dialogmote.domain.DialogmoteStatus
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.testhelper.UserConstants.ARBEIDSTAKER_FNR
+import no.nav.syfo.testhelper.generator.generateAvlysDialogmoteDTO
+import no.nav.syfo.testhelper.generator.generateNewDialogmoteDTO
+import no.nav.syfo.util.NAV_PERSONIDENT_HEADER
+import no.nav.syfo.util.bearerHeader
+import no.nav.syfo.varsel.MotedeltakerVarselType
+import no.nav.syfo.varsel.arbeidstaker.brukernotifikasjon.BrukernotifikasjonProducer
+import no.nav.syfo.varsel.arbeidstaker.domain.ArbeidstakerVarselDTO
+import org.amshove.kluent.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDateTime
+
+class ArbeidstakerVarselWhenMultipleDialogmoteApiSpek : Spek({
```

Comment:
Hva er det denne testen dekker som ikke allerede dekkes i `ArbeidstakerVarselApiSpek` ?
Om den dekker noe nytt, så foreslår jeg å  heller utvide `ArbeidstakerVarselApiSpek` slik at vi slipper dobbelt opp. Hver nye test som spinner opp hele testmiljøet tilfører også endel kjøretid ved testing.

Reply:
Den tester det tilfellet der det finnes mer enn et møte for den aktuelle personen. 

Skal sjekke om det er enkelt å slå sammen de to test-klassene.

Reply:
Det gikk greit det.